scripts/yggdrasil-iptunnel/install

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
+
+set -e
+
+BASE_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+
+sudo apt-get -y install jq
+
+sudo cp "$BASE_DIR/yggdrasil-setup" "/usr/local/sbin/yggdrasil-setup"
+sudo cp "$BASE_DIR/yggdrasil-pre-setup" "/usr/local/sbin/yggdrasil-pre-setup"
+
+sudo chmod a+x /usr/local/sbin/yggdrasil-pre-setup
+sudo chmod a+x /usr/local/sbin/yggdrasil-setup
+
+# Update service to start script on cjdns start
```

Comment:
cjdns or yggdrasil? I suggest searching for all instances of cjdns in this diff to make sure they are not misused for places where it should say yggdrasil.